addons/stock/migrations/10.0.1.1/post-migration.py

```diff
@@ -21,9 +21,9 @@ def update_picking_type_id(env):
     :param env: environment variable (self)
     """
     # load xml data to be used for filling in missing info
-    xml_stock_picking_type_int = env.ref("stock.picking_type_internal")
-    xml_stock_picking_type_out = env.ref("stock.picking_type_out")
-    xml_stock_picking_type_in = env.ref("stock.picking_type_in")
+    xml_stock_picking_type_int = env.ref("stock.picking_type_internal", False)
```

Comment:
If these refs are missing, then you have a serious problem... but the solution is not this, as later data will remain dangled.